pkg/storaged/stratis-panel.jsx

```diff
@@ -125,22 +125,14 @@ export function create_stratis_pool(client) {
                                { tag: "on", title: _("Encrypt data") }
                            ],
                            nested_fields: [
-                               PassInput("passphrase", _("Passphrase"),
-                                         {
-                                             validate: function (phrase) {
-                                                 if (phrase === "")
-                                                     return _("Passphrase cannot be empty");
-                                             },
-                                             visible: vals => vals.encrypt.on
-                                         }),
-                               PassInput("passphrase2", _("Confirm"),
-                                         {
-                                             validate: function (phrase2, vals) {
-                                                 if (phrase2 != vals.passphrase)
-                                                     return _("Passphrases do not match");
-                                             },
-                                             visible: vals => vals.encrypt.on
-                                         })
+                               NewPassInput("passphrase", _("Passphrase"),
+                                            {
+                                                validate: function (phrase) {
+                                                    if (phrase === "")
+                                                        return _("Passphrase cannot be empty");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18906-20230609-093742-36fdfe4a-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)